web/main.go

```diff
@@ -40,7 +40,7 @@ func main() {
 
 	installConfig, err := config.Install(pkgK8s.MountPathInstallConfig)
 	if err != nil {
-		log.Warnf("failed to load uuid from install config: %s", err)
+		log.Warnf("(disregard warning if running in development mode) failed to load uuid from install config: %s", err)
```

Comment:
This lgtm; was playing around with placing the explanation at the end, cause it feels slightly weird to me to have a warning that begins with "disregard warning".

TIOLI:
```
log.Warnf("failed to load uuid from install config: [%s] (disregard warning if running in development mode)", err)
```

Reply:
That's a good solution!